docs/_documentations/debugging.md

```diff
@@ -10,10 +10,41 @@ type: document
 
 # Debugging in Codewind
 
-Complete the following steps to enable debugging in Codewind applications:
+Complete the following steps to debug in Codewind applications:
 
 1. In the Codewind view, right-click a project and select **Restart in Debug Mode** from the menu.
 2. Wait for the project state to switch to **Debugging**. After it switches to Debugging mode, the debugger is attached.
 3. When in Debugging mode and when the debugger is attached, you can debug your application. Perform debug actions that are supported by the IDE that you are using.
 4. While the project is still in Debugging mode, if the debug process is detached, right-click your project and select **Attach Debugger** from the menu. This action reattaches the debugger to the debug process, and you can continue with debugging your application.
-5. To switch back to run mode, right-click your project and select **Restart in Run Mode** from the menu.
\ No newline at end of file
+5. To switch back to run mode, right-click your project and select **Restart in Run Mode** from the menu.
+
+## Debugging remotely
+Remote debugging is the same as local debugging with the following exceptions:
+- Install `kubectl` or `oc` and make sure you are logged in to your cluster.
+- In Eclipse, after the project restarts in debug mode, Codewind automatically sets up a port forward from the application's debug port to a port on your local machine.
+   - If you end the port forward and have debug sessions that depend on that port forward, those debug sessions also end.
+   - For Java projects, if you detach the debugger or restart the application in run mode, both the debug session and the port forward are terminated automatically.
```

Comment:
this is not quite true for vs code.
 If you close the debug session in vs code, the port-forward stays active. 
If you close the port forward, the debug session terminates because the debug port has closed.